java/F3DJavaBindings.cxx

```diff
@@ -0,0 +1,183 @@
+// Automatically generated headers
+#include <io_github_f3d_app_f3d_Camera.h>
+#include <io_github_f3d_app_f3d_Engine.h>
+#include <io_github_f3d_app_f3d_Loader.h>
+#include <io_github_f3d_app_f3d_Options.h>
+#include <io_github_f3d_app_f3d_Window.h>
```

Comment:
why would io_github be in the name of the header ?

Reply:
We have no choice, it's the JNI nomenclature

Reply:
Basically, the command `javac -h` internally called by CMake when `GENERATE_NATIVE_HEADERS` is enabled in the `add_jar` command is generating these files based on the Java package and classes names.

Reply:
fair enough, just wondering where it was coming from